src/google/cloud/happybase/table.py

```diff
@@ -969,3 +972,11 @@ def _row_keys_filter_helper(row_keys):
         return filters[0]
     else:
         return RowFilterUnion(filters=filters)
+
+
+def _get_row_set_object(row_start, row_stop):
+    """Return a RowSet object for the given row_start and row_stop
+    """
+    row_set = RowSet()
+    row_set.add_row_range(RowRange(row_start, row_stop))
```

Comment:
You could use [`row_set.add_row_range_from_keys()`](https://github.com/GoogleCloudPlatform/google-cloud-python/blob/e551423216cb8963262405de5be7d6851643f442/bigtable/google/cloud/bigtable/row_set.py#L69) here.

Reply:
Done, thanks.